deployment/terraform/step-functions/app_cli.json

```diff
@@ -0,0 +1,45 @@
+{
+    "StartAt": "Run scheduled task synchronously",
+    "States": {
+        "Run scheduled task synchronously": {
+            "Type": "Task",
+            "Resource": "arn:aws:states:::ecs:runTask.sync",
+            "Parameters": {
+                "Cluster": "${ecs_cluster_arn}",
+                "LaunchType": "FARGATE",
+                "NetworkConfiguration": {
+                    "AwsvpcConfiguration": {
+                        "SecurityGroups": [
+                            "${security_group_id}"
+                        ],
+                        "Subnets": ${subnet_ids}
+                    }
+                },
+                "TaskDefinition": "${ecs_task_definition_arn}",
+                "Overrides": {
+                    "ContainerOverrides": [
+                        {
+                            "Name": "${ecs_container_name}",
+                            "Command.$": "$.commands"
+                        }
+                    ]
+                }
+            },
+            "TimeoutSeconds": ${timeout_seconds},
+            "Catch": [
+                {
+                    "ErrorEquals": [
+                        "States.ALL"
+                    ],
+                    "Next": "Notify scheduled task failure"
+                }
+            ],
+            "End": true
+        },
+        "Notify scheduled task failure": {
+            "Type": "Task",
+            "Resource": "${lambda_function_arn}",
+            "End": true
+        }
```

Comment:
The [AWS docs](https://docs.aws.amazon.com/step-functions/latest/dg/connect-lambda.html) have some reference at the bottom on what I'm doing here.

> To invoke a Lambda function, you can also call the resource Amazon Resource Name (ARN) directly in the "Resource" string.

> The state input is passed to Lambda as the payload.